### DISCUSSION TITLE
CCU3 mega langsam und Ursache ReGaHss oder etwas anderes?

### DISCUSSION THREAD

**Discussion by Michael-Dreibholz** [OpenCCU/OpenCCU#1289] [🇩🇪 Fragen&Antworten]
@jens-maus,
sorry. Ich bin neu hier und vielen Dank für den Hinweis. 

Hallo zusammen,
ich bin langsam am Ende meiner Weisheit angekommen.
Bei mir wurde der Stromzähler getauscht und iobroker 
[CCU LOG.txt](https://github.com/jens-maus/RaspberryMatic/files/6599010/CCU.LOG.txt)
und CCU vorher runtergefahren.
Dann Neustart und seit diesem Zeitpunkt, ist meine CCU mega langsam nach Neustart und Bewegungsmelder usw. reagieren nicht mehr wirklich.
Was mir völlig unerklärlich ist, dass die Bewegungsmelder über einen DRI32 angeschlossen sind und nach geraumer Zeit in der CCU auf einen  Status "OFFEN" (Bewegung ausgelöst) springen, was vorher de facto nicht der Fall war.

Firmware auf allen Komponenten aktuell, Mediola Server, CUXD alles aktuell.
Iobroker alle Adapter aktuell.
Im Log habe 100te solcher Fehler stehen.

```
faultCode:-5,faultString:"Unknown Parameter value for value key: SET_POINT_TEMPERATURE"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]
Jun 4 15:00:59 CCU3 local0.err ReGaHss: ERROR: XMLRPC 'getValue' call failed (interface: 1011, params: {"000C9A49A7D34C:1","SET_POINT_TEMPERATURE"}) [CallGetValue():iseXmlRpc.cpp:1435]
Jun 4 15:00:59 CCU3 local0.err ReGaHss: ERROR: CallGetValue failed; sVal = 0.000000 [ReadValue():iseDOMdpHSS.cpp:124]
Jun 4 15:00:59 CCU3 local0.warn ReGaHss: WARNING: XMLRPC 'getValue': rpcClient.isFault() failed (url: xmlrpc://127.0.0.1:32010, params: {"000C9A49A7D34C:1","SWITCH_POINT_OCCURED"}, result: [faultCode:-5,faultString:"Unknown Parameter value for value key: SWITCH_POINT_OCCURED"]) [CallXmlrpcMethod():iseXmlRpc.cpp:2608]

```
Hat jemand einen Rat?

Vielen Dank für die Hilfe.

**Comment by jens-maus** [comment]
Tut mir leid, aber eine Rückfrage wäre erst einmal was du unter "mega langsam" verstehst? Schaltvorgänge dauern lange oder was meinst du damit? Auch schreibst du, das du ein HmIPW-DRI32 verwendest. Hast du die aktuellste HmIPW-DRAP firmware denn drauf? Denn die vorherigen Versionen hatten gewisse Probleme welche mitunter den DRAP in einen status gebracht das das dieser nicht mehr schnell oder korrekt funktionieren konnte.

**Comment by Michael-Dreibholz** [comment]
Es ist ein Hardwarethem und liegt vermutlich am DRI32 und das wird am Freitag hoffentlich behoben.
